framework/src/modules/chain/bft/bft.js

```diff
@@ -197,7 +199,8 @@ class BFT extends EventEmitter {
 		// Check BFT migration height
 		// https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#backwards-compatibility
 		const bftMigrationHeight =
-			this.constants.startingHeight - this.constants.activeDelegates * 2;
+			this.constants.startingHeight -
+			this.constants.activeDelegates * BFT_ROUND_THRESHOLD;
```

Comment:
This variable `bftMigrationHeight ` is used to initialize the `finalizedHeight `. This value is clearly documented in LIP to be 202 blocks from the height of migraiton. Can we discuss with science team before making this change. 

> we propose that chainMaxHeightFinalized is initialized with the value l-202

https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#backwards-compatibility